cmd/kyma/install/cmd.go

```diff
@@ -54,15 +52,15 @@ The standard installation uses the minimal configuration. The system performs th
 
 1. Deploys and configures the Kyma Installer. At this point, steps differ depending on the installation type.
 
-   When you install Kyma locally ` + "**from release**" + `, the system:
+When you install Kyma locally ` + "**from release**" + `, the system:
 
-   1. Fetches the latest or specified release along with configuration.
+1. Fetches the latest or specified release along with configuration.
```

Comment:
```suggestion
1.    Fetches the latest or specified release along with configuration.
```

I believe we need the indentation for proper formatting.